packages/web/src/components/payout-wallet-modal/PayoutWalletModal.tsx

```diff
@@ -99,7 +99,12 @@ const PayoutWalletModalForm = ({
           </Text>
           <RadioGroup {...optionField}>
             <Flex gap='l' direction='column'>
-              <Flex gap='m' alignItems='center'>
+              <Flex
+                gap='m'
+                alignItems='center'
+                as='label'
+                css={{ cursor: 'pointer' }}
```

Comment:
I am certain there's some type of a11y thing we're missing by doing it like this...

Reply:
@dylanjeffers wdyt?

Reply:
yeah good q, i think we are good cause we have the radio element inside of it, which covers the general a11y considerations